src/components/Home.jsx

```diff
@@ -1,14 +1,85 @@
 import React from 'react';
+import { ZooniverseLogotype, ZooniverseLogo } from 'zooniverse-react-components';
+import Divider from '../images/img_divider.png';
 
 class Home extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      backgroundHeight: 'auto',
+    };
+    this.resizeBackground = this.resizeBackground.bind(this);
+  }
+
+  componentDidMount() {
+    this.resizeBackground();
+    addEventListener('resize', this.resizeBackground);
+  }
+
+  componentWillUnmount() {
+    removeEventListener('resize', this.resizeBackground);
+  }
+
+  resizeBackground() {
+    const content = document.getElementById('home-logos');
+    if (content) {
+      const contentBottom = content.getBoundingClientRect().top;
+      const sectionHeight = document.body.scrollTop + contentBottom;
+      if (this.state.backgroundHeight !== sectionHeight) {
+        this.setState({ backgroundHeight: sectionHeight });
+      }
+    }
+  }
+
+  renderTopic() {
+    const numbers = ['One', 'Two', 'Three', 'Four'];
+    return numbers.map((num, i) => {
+      return <div key={i}>Topic {num} &#8608;</div>;
+    });
+  }
+
   render() {
     return (
       <main className="app-content home-page">
-        <b>Welcome to Liberating the Liberator.</b> Now let&apos;s drop some of that phat Lorem ipsum beat.
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eget eleifend risus, id aliquet est. Maecenas tempus luctus lacinia.
+        <div className="project-background" style={{ height: this.state.backgroundHeight }} />
```

Comment:
Yes, I think there might be a better way to have the project background included. This is what I was referring to earlier when I said there might be some messy code in this PR. In short, I'm trying to dynamically change the height of the background so something like the white line you mentioned below doesn't appear on resize.